src/GenerateEuler.m

```diff
@@ -0,0 +1,530 @@
+function GenerateEuler()

```

Comment:
Hey, my custom capitalized function names in Matlab let you know if the
function you're missing is in a toolbox you don't have or because of a path
problem.  Nothing to see there...

I've shifted this to a new dir gen.  We could also trash it - it needs
manual intervention for the 0.0 == -0.0, f(0.0) != f(-0.0) weirdness anyway

On 9 May 2016 at 17:37, Chris Foster <notifications@github.com> wrote:

> In src/GenerateEuler.m
> <https://github.com/FugroRoames/Rotations.jl/pull/2#discussion_r62458877>:
>
> > @@ -0,0 +1,530 @@
> > +function GenerateEuler()
>
> Eek, what can I say, it's matlab :-/
>
> Regardless of that unfortunate fact, let's put GenerateEuler.m somewhere
> other than src, since it's not actually part of the package source per se.
> perhaps gen/GenerateEuler.m?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/2/files/fb3a7882257b93afed97cfaa9bf56669bbf410dc#r62458877>
>